sdk/webpubsub/azure-messaging-webpubsubservice/examples/send_messages_aad_apim_proxy.py

```diff
@@ -0,0 +1,49 @@
+import io
+import logging
+import os
+
+from azure.messaging.webpubsubservice import WebPubSubServiceClient
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+# Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+# AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, WEBPUBSUB_ENDPOINT
+try:
+    endpoint = os.environ.get("WEBPUBSUB_ENDPOINT")
+    revers_proxy_endpoint = os.environ.get("WEBPUBSUB_REVERSE_RPOXY_ENDPOINT")
+except KeyError:
+    LOG.error("Missing environment variable 'WEBPUBSUB_ENDPOINT' or 'WEBPUBSUB_REVERSE_RPOXY_ENDPOINT' - please set if before running the example")
+    exit()
+
+# Build a client from the connection string. And for this example, we have enabled debug
+# tracing. For production code, this should be turned off.
+client = WebPubSubServiceClient(credential=DefaultAzureCredential(), endpoint=endpoint, revers_proxy_endpoint=revers_proxy_endpoint)
```

Comment:
should be `reverse_proxy_endpoint`. Can you run this sample to make sure that whatever's supposed to happen when you pass in `reverse_proxy_endpoint` actually happens? Thanks!

Reply:
If no `reverse_proxy_endpoint`, request will send to `endpoint`; if with `reverse_proxy_endpoint`, `reverse_proxy_endpoint` will forward request to `endpoint`. So the actual effect is the same. And I have corrected the name.